pkgs/by-name/tl/tlsinfo/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "tlsinfo";
+  version = "0.1.41";
+
+  src = fetchFromGitHub {
+    owner = "paepckehh";
+    repo = "tlsinfo";
+    rev = "v${version}";
```

Helps with branch/tag confusion.

```suggestion
    rev = "refs/tags/v${version}";
```